cstar/command.py

```diff
@@ -80,7 +80,7 @@ def _parse(command_name, filename, definition):
                 arguments.append(Argument(**json.loads(value)))
             else:
                 warn("Ignoring unknown property %s while parsing %s" % (name, filename))
-
+    
```

Comment:
Just a blank space, we could clean up.

Reply:
There's actually a few of these across the PR.

Reply:
ok, will fix this.